modules/tinymce/src/core/main/ts/api/dom/ScriptLoader.ts

```diff
@@ -243,6 +243,8 @@ class ScriptLoader {
           if (queue.length > 0) {
             self.queue = [];
             return load(queue).then(processResults);
+          } else {
+            return Promise.resolve();
```

Comment:
I'm surprised this was required, as returning `undefined` inside a promise has the same effect. I wonder if doing that explicitly would've been sufficient.